Automattic/wp-calypso
# PlansList: downgrade from 200GB for business plan description

With the storage downgrade of the business plan, we need to follow through with this change to the plan description.

HappyBlocks publishes the block `happy-blocks/pricing-plans` which is used on the support site and forums.

| Before  | After |
| ------------- | ------------- |
| <img width="909" alt="Screenshot 2023-07-20 at 14 05 15" src="https://github.com/Automattic/wp-calypso/assets/7000684/db7c179c-c864-493e-a25d-31ffa675f59f">  |  <img width="906" alt="Screenshot 2023-07-20 at 14 04 56" src="https://github.com/Automattic/wp-calypso/assets/7000684/acde95b9-ac90-48ae-861f-0660d57655f2"> |

## Proposed Changes

* Register `plans/updated-storage-labels` in happyblocks
* Use this flag to dynamically generate the description and short description of the business plan

## Testing Instructions
- Checkout this PR
- Run `cd apps/happy-blocks && yarn dev --sync`
- Sandbox `wordpress.com` and `en.support.wordpress.com`
- Visit https://wordpress.com/support/plan-features/business-plan/ without being logged in to WordPress
- The block description should say 50 instead of 200 GB
